criteria.md

```diff
@@ -0,0 +1,47 @@
+### Maturity Level 1 (Sandbox)
+ 
+| Category | Criteria Statement |
+|---|---|
+| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
+| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
+| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
+| Access Control | The project's version control system MUST prevent accidental deletion of the default branch. |
+| Build & Release | The project's build and release pipelines MUST NOT execute arbitrary code that is input from outside of the build script. |
+| Build & Release | All software releases MUST be assigned a standardized unique version identifier. |
+| Build & Release | Any websites related to the project development and release MUST be delivered using SSH or HTTPS and TLS. |
+| Documentation | The project MUST have one or more mechanisms for public discussions about proposed changes and usage obstacles. |
+| Documentation | The project documentation MUST provide user guides for all basic functionality. |
```

Comment:
What is basic functionality?  What is a user guide?

Reply:
Good questions to answer in a broader explanatory doc, along the lines of what [David Wheeler suggested](https://github.com/ossf/security-baseline/pull/24#issuecomment-2399993053), but not in the main table for readability.